backend/handlers/settings.go

```diff
@@ -182,46 +177,25 @@ func DeleteAccountDetails(postgresDb *gorm.DB) func(w http.ResponseWriter, r *ht
 
 func Rescan(postgresDb *gorm.DB) func(w http.ResponseWriter, r *http.Request) {
```

Comment:
can we create a different struct with just AccountName, so there's not confusion about the other fields of AccountDetails not being present